applications/CoSimulationApplication/custom_data_structure/pyKratos/TriangleElement.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function, absolute_import, division
+from numpy import *
+from .Element import *
+
+
+class Triangle(Element):
+
+    def __init__(self, node_list):
```

Comment:
Don't you need to call the base class constructor in here?

Reply:
done